rainier-notebook/src/main/scala/com/stripe/rainier/notebook/package.scala

```diff
@@ -56,24 +55,6 @@ object Jupyter {
   implicit val extent: Extent = Extent(400, 400)
   implicit val theme: Theme = PlotThemes.default
 
-  def trace(t: Trace)(implicit
```

Comment:
This was not a great implementation of `trace` anyway. We can bring back something better.